src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -78,23 +81,43 @@ public void resetData(ReadOnlyAddressBook newData) {
     }
 
     /**
-     * Returns {@UniquePersonList} of all blacklisted persons
-     * in the existing data of this {@code AddressBook} with {@code newData}.
+     * Searches for blacklisted persons in {@code persons}.
+     * @return {@code UniquePersonList} of all blacklisted person in {@code persons}.
      */
     public UniquePersonList getBlacklistedPersons() {
         UniquePersonList blacklistedPersons = new UniquePersonList();
         for (Person person : persons.getInternalList()) {
-            if (person.getIsBlacklisted()) {
+            if (person.isBlacklisted()) {
                 try {
                     blacklistedPersons.add(person);
                 } catch (DuplicatePersonException e) {
-                    assert false : "This is not possible as prior checks have been done";
+                    assert false : "This is not possible as prior checks have"
+                            + " been done to ensure AddressBook does not have duplicate persons";
                 }
             }
         }
         return blacklistedPersons;
     }
 
+    /**
+     * Searches for whitelisted persons in {@code persons}.
+     * @return {@code UniquePersonList} of all whitelisted persons in {@code persons}.
```

Comment:
i think the variable after `@return` does not need to be in `{@code ...}`

Reply:
Noted